### ISSUE TITLE
option the launch crusader and crusader extreme from within the UCP patcher after finishing the install.

### ISSUE THREAD

**Issue by CrynCara** [Sh0wdown/UnofficialCrusaderPatch#677]
I had a user today that had problems with the ucp.

he tried to click the "Fertig"/(finished) that the UCP shows after the install is concluded to launch the game.
Doesn't work currently but could be a nice feature - would as well stop users from forgetting where they installed the UCP.

**Comment by Truetobi91** [comment]
Yeah I remember trying to click that 20 times the first time of using the launcher xD
